src/matcha_ml/cli/analytics.py

```diff
@@ -5,7 +5,7 @@
 from matcha_ml.services.global_parameters_service import GlobalParameters
 
 # create a typer app to group all analytics subcommands
-app = typer.Typer()
+app = typer.Typer(no_args_is_help=True)
```

Comment:
For consistency with main app in `cli.py`, we can disable `pretty_exceptions_show_locals` as well inside the Typer.

Reply:
Added, thanks! ➕